packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.test.tsx

```diff
@@ -179,3 +181,27 @@ it('should default to light theme if no parent theme found', () => {
   render(<ThemeProvider>Test</ThemeProvider>);
   expect(screen.getByText('Test')).toHaveAttribute('data-iui-theme', 'light');
 });
+
+it('should respect the portalContainer prop', async () => {
+  const myPortals = document.createElement('my-portals');
+  document.body.appendChild(myPortals);
+
+  const InPortal = () => {
+    const { portalContainer } = React.useContext(ThemeContext) || {};
+    return (
+      portalContainer &&
+      ReactDOM.createPortal(<div>in portal!</div>, portalContainer)
+    );
+  };
+
+  const { getByText } = render(
+    <ThemeProvider portalContainer={myPortals}>
+      <div>not in portal</div>
+      <InPortal />
+    </ThemeProvider>,
+  );
+
+  getByText('not in portal');
```

Comment:
In mulitple `.test.tsx` files, I am noticing this pattern of `getByText(...)`'s return value not being stored in a variable. Is this just a way to make sure that there is a node with textContent of `"not in portal"` but not do anything with it?

Similarly, is L204 the shorthand for:
```tsx
expect(getByText('not in portal').toBeTruthy())
```

Reply:
yeah pretty much. it will throw an error if the text is not found